test_collections/sdk_tests/support/yaml_tests/sdk_yaml_tests.py

```diff
@@ -108,10 +112,35 @@ def sdk_yaml_test_collection(
 
     files = yaml_test_folder.yaml_file_paths()
     version = yaml_test_folder.version
-    suites = _parse_all_sdk_yaml(yaml_files=files, yaml_version=version)
+    suites = _parse_all_yaml(yaml_files=files, yaml_version=version)
 
     for suite in suites:
         suite.sort_test_cases()
         collection.add_test_suite(suite)
 
     return collection
+
+
+def custom_yaml_test_collection(
+    yaml_test_folder: YamlTestFolder = CUSTOM_YAML_TEST_FOLDER,
+) -> YamlCollectionDeclaration:
+    """Declare a new collection of test suites with the 3 test suites."""
```

Comment:
```suggestion
    """Declare a new collection of test suites."""
```
At this moment we still don't know how many test suites will be right?